signal-slot-benchmarks/vs/signal-slot-benchmarks.vcxproj

```diff
@@ -237,7 +237,7 @@
     <ClInclude Include="..\benchmark\hpp\benchmark_css.hpp" />
     <ClInclude Include="..\benchmark\hpp\benchmark_dob.hpp" />
     <ClInclude Include="..\benchmark\hpp\benchmark_evl.hpp" />
-    <ClInclude Include="..\benchmark\hpp\benchmark_fts.hpp" />
```

Comment:
why removed?

Reply:
I just copied the file from upstream. I fix it.